zombienet-sdk-tests/tps/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
```

Comment:
Why not including this folder inside the workspace and adding all these dependencies to the main `Cargo.toml`? This sub-project does not need to be enabled by default by setting `default-members`.

Reply:
Sounds good to me, ping @s0me0ne-unkn0wn ?

Reply:
Otherwise this may become the typical folder that gets completely outdated and whenever we are ready for a release it so happens that it does not work, since it was not frequently built in the CI and errors detected on time.